spec/board_spec.rb

```diff
@@ -1,49 +1,19 @@
-require './lib/board'
+require './lib/board.rb'
 
-RSpec.describe 'Board methods' do
-  output = "+------------------------+\n"
-
-  9.times do |num|
-    output += '|' if (num + 1) == 1 || (num + 1) == 4 || num + 1 == 7
-    output += " #{num + 1} - \u{2754} "
-    output += "| \n" if (num + 1) % 3 == 0
-  end
-
-  output += '+------------------------+'
-
-  # updated board for x
-
-  updated_board_x = "+------------------------+\n"
-
-  9.times do |num|
-    updated_board_x += '|' if (num + 1) == 1 || (num + 1) == 4 || num + 1 == 7
-    updated_board_x += (num + 1) == 5 ? " #{num + 1} - \u{274C} " : " #{num + 1} - \u{2754} "
-    updated_board_x += "| \n" if (num + 1) % 3 == 0
+RSpec.describe 'testing the board' do
+  it 'should show us an empty board' do
+    expect(Board.new.show_board).to eq("+------------------------+\n| 1 - ❔  2 - ❔  3 - ❔ | \n| 4 - ❔  5 - ❔  6 - ❔ | \n| 7 - ❔  8 - ❔  9 - ❔ | \n+------------------------+")
```

Comment:
- [ ] Please add a spec that checks for negative scenarios / incorrect usage of the method. 
  _For example, here you can add a scenario where you expect Board.new.show_board to not equal a board with characters_